auxx/src/Lang/Lexer.hs

```diff
@@ -216,9 +216,9 @@ pAddress = do
 
 pPublicKey :: Lexer PublicKey
 pPublicKey = do
-    str <- pSomeAlphaNum
+    str <- (<>) <$> takeP (Just "base64") 86 <*> string "=="
```

Comment:
I don't get this line, can you clarify?

Reply:
Read 86 symbols that end with `==`, then we parse it as base64. I tested

Reply:
Ah, I forgot that public key has fixed length, ok then.